docs/FAQ/index.md

```diff
@@ -28,135 +36,138 @@ directory with product configuration and user groups in the Admin Console
+
+### If something changes in the API (new field in creating users, for example ) how will the update  be applied to the user sync tool?
+
+User sync is an open source project.  Users can download and build the latest 
 sources at their discretion.  Adobe will post new releases with builds periodically.  
 Users can stay informed of them via git notifications.  When adopting a new release, 
 only the single pex file needs to be updated by the user.  If there are configuration 
 changes or command line changes to support new features, there may be updates in 
 those files to take advantage of them.
 
-**Q: Does User sync need some sort of whitelisting with the firewall rules of the machine
-on which it runs?** I see Windows users are more prone to such errors, from support 
-experience.<br/>
-A: User sync needs to be able to access the enterprise directory system and Adobe.  
+### Does User sync need some sort of whitelisting with the firewall rules of the machine
+on which it runs? I see Windows users are more prone to such errors, from support 
+experience.
+
+User sync needs to be able to access the enterprise directory system and Adobe.  
 Since there can be significant variance across system setups, we can't give any 
 specific steps.  LDAP typically uses port 389, but can be configured by IT.  Adobe 
 only requires REST calls over SSL on port 443.
 
-**Q: There is something specified about a second step in differentiating the users 
+### There is something specified about a second step in differentiating the users 
 of a bigger company by adding the country code and the cost centre as metadata 
-to users when they are created; is user sync set up with this feature or not 
-this version?**<br/>
-A: Country is required for Federated Ids and useful for Enterprise Ids (if not 
+to users when they are created; is user sync set up with this feature?
+
+Country is required for Federated Ids and useful for Enterprise Ids (if not 
 supplied, the user will have to supply the country on first login).  User Sync 
 supports using the standard control field from LDAP, and also supports the 
 specification of a default country code to be used.  
 User Sync can also be customized to call a customer-provided python "hook" in the 
 User Sync configuration; this hook can use customer-side attributes such as 
 cost center to control attributes and group mappings of users on the Adobe side.
  
-**Q: Is User Sync part of Adobe's offering to EVIP customers as well as ETLA customers?** 
-Or to put it another way, do EVIP customers have access to the UM API?<br/>
-A: Yes, EVIP customers have access to the UMAPI and User Sync.
+### Is User Sync part of Adobe's offering to EVIP customers as well as ETLA customers?
+ 
+Or to put it another way, do EVIP customers have access to the UM API?
+
+Yes, EVIP customers have access to the UMAPI and User Sync.
```

Comment:
I suggest "Yes, all Enterprise customers have access to the UMAPI and User Sync, regardless of their buying program (E-VIP, ETLA, or Enterprise Agreement).